packages/plugin-simple-throttle/throttle.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     maxEvents: {
       defaultValue: () => 10,
       message: 'should be a positive integer ≤100',
-      validate: val => positiveIntIfDefined(val) && val < 100
+      validate: val => intRange(1, 100)(val)
```

Comment:
I believe that <100 has changed to <= 100 here, is that intentional/does it matter?

Reply:
it now matches the explanation in the message above!